src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructure.java

```diff
@@ -996,7 +998,11 @@ private boolean placeBlock(@NotNull final BlockPos pos, final Block block, @NotN
             final List<ItemStack> items = BlockPosUtil.getBlockDrops(world, pos, 0);
             for (final ItemStack item : items)
             {
-                InventoryUtils.setStack(worker.getInventoryCitizen(), item);
+                ItemStack itemRemaining = ItemHandlerHelper.insertItemStacked(worker.getInventoryCitizen(), item, false);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Local variable 'itemRemaining' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)